contracts/staking/src/contract.rs

```diff
@@ -321,6 +325,10 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, C
 
     // set new contract version
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+
+    // update v2 oracle contract address
+    update_config_from_migrate(deps, msg)?;
```

Comment:
I would suggest first updating the configuration, and then setting the new contract version. This ensures that the configuration migration is part of the `migrate data` step (currently void) and that all the data has been migrated properly before setting the new contract version. 

Theorically it shouldn't change much, but logically it makes more sense to first migrate the data and then set the version, just in case anything else, to avoid having a state in which the contract has the new version but some data are not yet migrated.